components/alert/AlertMap.vue

```diff
@@ -711,250 +867,181 @@ export default Vue.extend({
     },
     async loadRegions(): Promise<void> {
       if (this.mapRegions.size !== 0) {
-        return;
+        return
       }
 
-      const regions = await new MapRegionDataRequest()
-        .pull(this.alert.zone ? this.alert.zone : Zone.INDAR);
-      
-      this.$L.Icon.Default.prototype.options.iconAnchor = [0, 0];
-      this.$L.Icon.Default.prototype.options.tooltipAnchor = [0, 0];
+      const regions = await new MapRegionDataRequest().pull(
+        this.alert.zone ? this.alert.zone : Zone.INDAR
+      )
+
+      this.$L.Icon.Default.prototype.options.iconAnchor = [0, 0]
+      this.$L.Icon.Default.prototype.options.tooltipAnchor = [0, 0]
```

Comment:
Uhhhh wut? Please explain 😕

Reply:
I think this was a leftover from when I was still messing around with leaflet stuff, removed it and nothing broke so 🤷‍♂️